code/datums/components/electrified.dm

```diff
@@ -0,0 +1,279 @@
+GLOBAL_LIST_INIT(unelectrifiable_machines, typecacheof(list(/obj/machinery/door/airlock/cult)))

```

Comment:
Why does this need to exist ? Wouldn't this make more sense as can electrify signal ?